docs/deployment/flink/basics.md

```diff
@@ -0,0 +1,92 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<div align=center>
+
+![](../../imgs/kyuubi_logo.png)
+
+</div>
+
+# Basics
```

Comment:
IMO, the `basic.md` is unnecessary for the Flink engine users because the users should have the knowledge of the Flink configuration.

Reply:
This document ensures that users can absolutely run a Flink job. The vast majority of users who read documents are not so familiar with Flink and Kyuubi. This document can be displayed directly when we answer questions in the community. Users can retrieve all knowledge through it. A novice of Flink and Kyuubi can jump to other required information through this document.

Reply:
Users often forget to configure `execution.target`, resulting in the running result of Kyuubi not meeting expectations.

Reply:
In the future, we make a FlinkSQLEngine Q&A docs about the `execution.target` option. Now, I delete `flink/basics.md`.